appkit/src/test/scala/org/ergoplatform/appkit/AnonymousAccessSpec.scala

```diff
@@ -127,7 +127,7 @@ object DhtUtils {
     val tx = BoxOperations.createForSender(sender.getAddress, ctx).buildTxWithDefaultInputs { txB: UnsignedTransactionBuilder =>
       val outBox = txB.outBoxBuilder
         .value(dhtBox.getValue)
-        .contract(new ErgoTreeContract(receiver.getErgoAddress.script, receiver.getNetworkType))
+        .contract(receiver.getErgoContract)
```

Comment:
Better name would be `asErgoContract` or `toErgoContract`, because Address and ErgoContract are isomorphic. Basically two different representations of the same thing.